lib/psabpf_table.c

```diff
@@ -959,96 +1068,60 @@ static int construct_buffer(char * buffer, size_t buffer_len,
     return return_code;
 }
 
-static bool member_is_counter(psabpf_table_entry_ctx_t *ctx, const struct btf_member *member)
+static int handle_direct_objects(const char *key, char *value,
```

Comment:
I suggest to create separated methods for counters and meters in `handle_direct_objects`

Reply:
Done